styles/main.css

```diff
@@ -518,3 +529,206 @@ form ul li textarea:focus {
+  margin-bottom: 20px;
+}
+
+.content {
+  display: grid;
+  grid-template-rows: auto auto;
+  padding: 20px 0;
+  background-color: #fff;
+  border: 1px solid #dfe1e6;
+  border-radius: 9px;
+  margin-bottom: 100px;
+}
+
+.snapshoot img {
+  width: 100%;
+}
+
+.text-content {
+  margin-bottom: 40px;
+}
+
+.tonic {
+  padding: 5px 0 5px 20px;
+  font-style: normal;
+  font-weight: bold;
+  font-size: 32px;
+  line-height: 44px;
+  color: #172b4d;
+}
+
+.detail {
+  padding: 0 20px 0 20px;
+  font-style: normal;
+  font-weight: normal;
+  font-size: 15px;
+  line-height: 24px;
+}
+
+.profile {
+  display: flex;
+  align-items: center;
+  gap: 20px;
+  padding-left: 20px;
+}
+
+.back-end {
+  font-weight: 600;
+  font-size: 13px;
+  line-height: 16px;
+  color: #7a869a;
+}
+
+.canopy {
+  font-weight: 600;
+  font-size: 13px;
+  line-height: 16px;
+  text-transform: uppercase;
+  color: #344563;
+}
+
+.tools {
+  display: flex;
+  gap: 20px;
+  padding-left: 20px;
+}
+
+.tool {
+  background: #ebebff;
+  border-radius: 8px;
+  padding: 4px 12px;
+  color: #396df2;
+}
+
+.see-project {
+  font-weight: 500;
+  font-size: 17px;
+  line-height: 24px;
+  margin-left: 20px;
+  color: #396df2;
+  padding: 12px;
+  border: 1px solid #6070ff;
+  box-sizing: border-box;
+  border-radius: 8px;
+  background-color: #fff;
+}
+
+/* SECTION DETAIL */
+
+/* MOBILE */
+
+.modal {
+  display: grid;
+  grid-template-rows: auto auto;
+  background-color: #fefefe;
+  margin: 15% auto; /* 15% from the top and centered */
+  padding: 20px;
+  border: 1px solid #888;
+  width: 80%;
+  border-radius: 10px;
+}
+
+.modal-div-1 {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  margin: 0 21px;
+}
+
+.mob {
+  display: flex;
+}
+
+.modal-title {
+  font-style: normal;
+  font-weight: bold;
+  font-size: 32px;
+  line-height: 44px;
+  color: #172b4d;
+  margin: 0;
+}
+
+.close-modal {
+  font-size: 30px;
+  color: #67798e;
+}
+
+.modal-div-2 {
+  display: flex;
+  gap: 20px;
+  margin: 0 21px;
+}
+
+.modal-img {
+  margin: 0 21px;
+}
+
+.snapshoot1-mob {
+  width: 100%;
+}
+
+.snapshoot1-desk {
+  width: 100%;
+  display: none;
+}
+
+.modal-info {
+  margin: 0 21px;
+}
+
+.modal-tools {
+  display: flex;
+  gap: 20px;
+  padding-left: 0;
+  border-bottom: 1px solid #ebecf0;
+  margin-top: 20px;
+}
+
+.modal-btn {
+  font-weight: 500;
+  font-size: 17px;
+  line-height: 24px;
+  color: #396df2;
+  padding: 12px;
+  border: 1px solid #6070ff;
+  box-sizing: border-box;
+  border-radius: 8px;
+  background-color: #fff;
+  margin: 20px 20px 0 0;
+}
+
+.modal-btn img {
+  margin-left: 5px;
+}
+
+/* MODAL */
+
+.popup {
+  display: none;
+  position: fixed;
+  z-index: 1; /* Sit on top */
+  left: 0;
+  top: 0;
+  width: 100%; /* Full width */
+  height: 100%; /* Full height */
+  overflow: auto; /* Enable scroll if needed */
+  background-color: rgb(0 0 0); /* Fallback color */
+  background-color: rgb(0 0 0 / 40%); /* Black w/ opacity */
+}
```

Comment:
- [OPTIONAL ] The popup follows the design for both desktop and mobile versions from [the chosen template in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1), 
- [OPTIONAL] Kindly consider also adding these icons  the `See live` and `See source` buttons of your popup window :arrow_down: 


See  | Icons
------------- | -------------
![lasttttt](https://user-images.githubusercontent.com/57562869/152654762-55479d0a-7716-448f-9cad-549f4b9bcbb5.png)  | ![lastt2](https://user-images.githubusercontent.com/57562869/152654774-5f486c07-fa73-4903-9b14-0cd6897eed15.png)

Reply:
Thank you so much, I'll fix them